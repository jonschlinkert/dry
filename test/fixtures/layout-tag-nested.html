{% layout "layout-nested.html" %}

This is root-text-node content from "fixtures/layout-nested.html".

{% block "footer" %}
footer: "fixtures/layout-nested.html"
{% endblock %}
