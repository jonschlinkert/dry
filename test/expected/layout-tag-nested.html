<!DOCTYPE html>
<html lang="en">
  
  parent: I should be rendered
  
  

  

<head>
  <meta charset="UTF-8">
  <title>Body fixture</title>
</head>
<body>
  
This is root-text-node content from "fixtures/layout-nested.html".

  
footer: "fixtures/layout-nested.html"
</body>
  

</html>